runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/io/UnboundedSocketSource.java

```diff
@@ -92,27 +95,30 @@ public long getDelayBetweenRetries() {
   }
 
   @Override
-  public List<? extends UnboundedSource<String, C>> generateInitialSplits(int desiredNumSplits, PipelineOptions options) throws Exception {
-    return Collections.<UnboundedSource<String, C>>singletonList(this);
+  public List<? extends UnboundedSource<String, CheckpointT>> generateInitialSplits(
+      int desiredNumSplits, PipelineOptions options) throws Exception {
+    return Collections.<UnboundedSource<String, CheckpointT>>singletonList(this);
   }
 
   @Override
-  public UnboundedReader<String> createReader(PipelineOptions options, @Nullable C checkpointMark) {
+  public UnboundedReader<String> createReader(PipelineOptions options, @Nullable CheckpointT
+      checkpointMark) {
     return new UnboundedSocketReader(this);
   }
 
   @Nullable
   @Override
   public Coder getCheckpointMarkCoder() {
-    // Flink and Dataflow have different checkpointing mechanisms.
+    // Flink and Beam have different checkpointing mechanisms.
     // In our case we do not need a coder.
     return null;
   }
 
   @Override
   public void validate() {
     checkArgument(port > 0 && port < 65536, "port is out of range");
-    checkArgument(maxNumRetries >= -1, "maxNumRetries must be zero or larger (num retries), or -1 (infinite retries)");
+    checkArgument(maxNumRetries >= -1, "maxNumRetries must be zero or larger (num retries), or -1"
+        + " (infinite retries)");
```

Comment:
Another option would be to move the entire string to the next line and keep it together:

```
checkArgument(maxNumRetries >= -1,
    "maxNumRetries must be zero or larger (num retries), or -1 (infinite retries)");
```